js/index.js

```diff
@@ -1,135 +1,104 @@
-		// Reorder array in random sequence
-		let randomlyOrderedArray = arr.sort(function() { return .5 - Math.random(); });
-		// Take re-ordered array and only keep as many as needed
-		let newArray = randomlyOrderedArray.slice(0, days);
-		callback(newArray);
-	}
-
-	// Create HTML list elements and inject into DOM
-	const createListHTML = function createListHTML(arr) {
-		// Create html elems
-		let container = document.createElement('section'),
-				main = document.querySelector('main'),
-				list = document.createElement('ul'),
-				btnWrapper = document.createElement('div');
-
-		// Add props to elems
-		container.className = 'content-container --no-border';
-		list.className = 'list js-list';
-		btnWrapper.className = 'btn-wrapper';
-
-		// Build the list
-		main.appendChild(container);
-		container.appendChild(list);
-		container.appendChild(btnWrapper);
-
-		populateList(arr);
-	} 
-	
-	// Create list items from array and inject into DOM
-	const populateList = function populateList(arr) {	
-		let menuList = document.querySelector('ul');
-
-		arr.forEach(function(item) {
-			// Create list html elems
-			let	link = document.createElement('a'),
-					linkItem = document.createElement('li'),
-					removeBtn = document.createElement('button');
-		
-			// Add props to elems
-			link.innerHTML = item.name;
-			link.className = 'js-link-item'
-			removeBtn.className = 'secondary-btn kill-item-btn js-remove';
-			removeBtn.innerHTML = 'x';
-
-			// Build the list item
-			linkItem.appendChild(link);
-			linkItem.appendChild(removeBtn);
-			menuList.appendChild(linkItem);
-		});
-	}
-
-	// Remove list from DOM
-	const removeList = function removeList() {
-		let listContainer = document.querySelectorAll('.content-container')[1];
-		listContainer.remove();
-	}
-
-	// Remove individual list item
-	const removeListItem = function removeListItem(e) {
-		let listItem = e.target.closest('li');		
-		listItem.remove();
-	}
-	
-	const refreshItem = function refreshItem(arr) {
-		// compare arrays and return unrepeated obj to replace current
-	}
-
-	// const showDetails = function showDetails(e) {
-	// 	let listItem = this.closest('li');
-	// }
-	// 	const myMeals = Meals.filter(function(whom){ return whom.owner === 'Brian'}).map(function(whom){ return whom.name; });
-
+(function () {
+    'use strict';
+
+    const GENERATE_BTN = document.querySelector('#GenerateMealList');
+    const URL = 'http://www.thebrianpye.com/Recipes/meals.json';
+    const BTN_TEXT = [
+        `Make it again`,
+        `Nope, try again`,
+        `Keep on trying`,
+        `I Don't like this` //Template literals preserve whitespace and quotation marks.
+    ];
+
+    let count = 0;
+    GENERATE_BTN.addEventListener('click', (ev) => main(ev));
+
+
+    function main(ev) {
+        const listContainer = document.querySelectorAll('.content-container')[1]; //I tried setting it to query an id, but I kept getting undefined errors for some reason. Leaving it alone for now.
```

Comment:
No worries here - I updated the selector by adding a specific `js-` prefixed class